### DISCUSSION TITLE
Is there a guide on how to setup a Wallet, a Full Node, a Farmer and multiple Harvesters under Linux?

### DISCUSSION THREAD

**Discussion by usao-az** [Chia-Network/chia-blockchain#4491] [Q&A]
Is there a guide on how to setup a Wallet, a Full Node, a Farmer and multiple Harvesters under Linux?

**Comment by ChesDonno** [comment]
https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL

This what you're looking for?

**Reply by usao-az** [reply]
Ok, went through the document, and already have done all those steps. What's not described is specifically how to configure a Wallet, a Full Node a Farmer and multiple Harvesters. I have the software installed and working in Linux, but I don't know how to start/stop the various pieces nor how to get the pieces to talk to eachother.

**Reply by retrominer** [reply]
Hi,
You can setup multiple Farmers look at this Wiki.
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

**Reply by usao-az** [reply]
Thanks, but is there anything describing how to setup the main node? I think that has to be first.
The main node is just the wallet, correct? Doesn't need to see any plots?
I don't think the Farmer needs to see the plots either, so I was thinking of combining these on a VM.

**Reply by LDizon** [reply]
You don't have to do anything as involved as you did when you set up a Harvester to run the other Chia services. Farmer, harvester, a full node, and the wallet are all separate. See the section about the `start` command here: https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference
